languages/sardinian/pack/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,31 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.anysoftkeyboard.languagepack.sardinian.pack">
+
+    <application>
+        <receiver
+            android:name="com.anysoftkeyboard.languagepack.sardinian.PackBroadcastReceiver"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="com.menny.android.anysoftkeyboard.KEYBOARD" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.ALTERNATIVE" />
+                <category android:name="android.intent.category.SELECTED_ALTERNATIVE" />
+            </intent-filter>
+            <intent-filter>
+                <action android:name="com.menny.android.anysoftkeyboard.DICTIONARY" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.ALTERNATIVE" />
+                <category android:name="android.intent.category.SELECTED_ALTERNATIVE" />
+            </intent-filter>
+
+            <meta-data
+                android:name="com.menny.android.anysoftkeyboard.keyboards"
+                android:resource="@xml/sardinian_keyboards" />
+            <meta-data
+                android:name="com.menny.android.anysoftkeyboard.dictionaries"
+                android:resource="@xml/sardinian_dictionaries" />
```

Comment:
this XML is also missing.

Reply:
Same as the keyboards one, I've added it right now.